### ISSUE TITLE
Get timer includes `tz`

### ISSUE THREAD

**Issue by scrool** [scrool/xled-docs#26]
At least in FW version 2.8.3 [get timer](https://xled-docs.readthedocs.io/en/latest/rest_api.html#get-timer) includes key `tz` that seems to be a definition of a timezone in unknown format. E.g. for CET/CEST timezone:
```
"tz": "CET-1CEST,M3.5.0,M10.5.0/3",
```

**Comment by JLOrwell** [comment]
This is true.  The set function also has it.  However it doesn't seem to be required since you're also required to set the time when you use the function.  That is an openwrt string.
